pom.xml

```diff
@@ -56,10 +56,10 @@ THE SOFTWARE.
       It declares version of the core, which should be included into the the server-side.
       Note that the plugin still bundles agent support for Maven 3.0.5, probably it's not really operational
     -->
-    <mavenMinimalVersion>3.5.4</mavenMinimalVersion>
+    <mavenMinimalVersion>3.8.1</mavenMinimalVersion>
```

Comment:
So this controls https://github.com/jenkinsci/maven-plugin/blob/7a04dcdd358bbbfde15ee8766b9baccdf3f32be3/pom.xml#L405-L468 but then what are `mavenVersion` / `maven.version`? These POM properties do not seem to be used.

Reply:
garbage I guess :) We can address this with another change